.changeset/thin-pants-turn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@aws-amplify/ui-react-ai": patch
+---
+
+fix(ai): fixes reattaching the same file by always triggering the onC…
```

Comment:
fix?

Reply:
Sorry I mean the `the onC…` at the end of the line, should probably finish the sentence?

Reply:
oh haha i didnt even notice that